src/translations/en.json

```diff
@@ -32,7 +32,7 @@
         "joinMail": "Join Mailing List",
         "donate": "Donate",
         "returnFirstStep": "Return to First Step",
-        "systemDetails": "System Details",
```

Comment:
We have also system, no need to change this. I can update it as System later, but not this way. Can you rollback this change? Then I apply your changes.